src/cycax/cycad/features.py

```diff
@@ -141,13 +141,20 @@ def swap_yz(self, rot: float, rotmax: list) -> list:
 nut_specifications: dict[
```

Comment:
Why is this global? Could it not just be part of nut object? Not serious just wondering if there is a good reason.